### PR TITLE
Move definition of celery beat schedule

### PR DESCRIPTION
**Pull request by annehaley**
The demo instance hasn't been running the reset task. Moving the definition fixed this in a dev env.

### CODE REVIEW

File: miqa/settings.py [hunk 1]
```diff
@@ -46,7 +46,17 @@ class MiqaMixin(ConfigMixin):
...
     ACCOUNT_FORMS = {'signup': 'miqa.core.rest.accounts.AccountSignupForm'}
 
-    CELERY_BEAT_SCHEDULE = {}
+    @property
+    def CELERY_BEAT_SCHEDULE(self):
+        if self.DEMO_MODE:
+            return {
+                'reset-demo': {
+                    'task': 'miqa.core.tasks.reset_demo',
+                    'schedule': timedelta(days=1),
```
Comment (zachmullen) [inline miqa/settings.py:55]: This also changes from `crontab` to `timedelta` -- is that a necessary part of this fix?

Comment (annehaley) [reply miqa/settings.py:55]: I couldn't get the task to run with crontab; I only observed it working with timedelta

**Comment by annehaley** [comment]
Also, do you know why the lint test is getting the current error?

**Comment by zachmullen** [comment]
No idea, but I assume it's our use of cache-tox in CI. Maybe try removing or upgrading that.

**Review by zachmullen — APPROVED** [review]

**Pull request merged**